src/autocomplete/databases/clickhouse/index.ts

```diff
@@ -0,0 +1,53 @@
+import {
+    AutocompleteResultBase,
+    CursorPosition,
+    EngineSuggestion,
+    TableOrViewSuggestion,
+} from '../../shared/autocomplete-types';
+import {clickHouseAutocompleteData} from './clickhouse-autocomplete';
+import {parseQuery, parseQueryWithoutCursor} from '../../shared/autocomplete';
+import {separateQueryAndCursor} from '../../shared/parse-query-with-cursor';
+
+export {ClickHouseLexer} from './generated/ClickHouseLexer';
+
+export {ClickHouseParser} from './generated/ClickHouseParser';
```

Comment:
let's not export this
we don't have a usecase for now

Reply:
Files are very useful for highlighting semantics errors...Lexer and ...Parser

Reply:
For error highlight you may use `parsePostgreSqlQueryWithoutCursor(query).errors`

Reply:
better to just use `parseClickHouseQuery`

Reply:
yes, you're right 

in this case, I can use `parsePostgreSqlQueryWithoutCursor`

deleted